server/dbconnection/connection.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const mongouri = 'mongodb://localhost:27017/yum_eats'; // create schema before running
+
+const db = mongoose.connect(mongouri,{
+	useNewUrlParser : true
```

Comment:
What does this useNewUrlParser do?  I've never seen it before.

Reply:
forgot to answer this, but this just turns off the DeprecationWarning really. With out it it would just print _"DeprecationWarning: current URL string parser is deprecated, and will be
removed in a future version. To use the new parser, pass option
{ useNewUrlParser: true } to MongoClient.connect."_ in your console.